File: examples/before/sample.js
// Example JavaScript code with deprecated API
const oldApi = require('old-api');
const { getUser, fetchPosts } = require('old-api');

function main() {
    const client = oldApi.createClient({ key: 'secret' });
    
    // Direct call
    const user = oldApi.getUser(123);
    console.log(user);
    
    // Destructured call
    const posts = fetchPosts(123);
    
    // Client instance call
    const data = client.fetchPosts(123);
    
    return { user, posts, data };
}

main();

File: examples/before/sample.py
# Example Python code with deprecated API
from old_api import Client, fetch_data
import old_api as api

def main():
    client = Client(api_key="secret")
    
    # Direct call
    user = old_api.get_user(user_id=123)
    print(user)
    
    # Call via client instance (if client uses same method names)
    posts = client.fetch_posts(user_id=123)
    
    # Function import call
    data = fetch_data(user_id=123)
    
    # Alias call
    result = api.get_user(user_id=456)
    
    return user, posts, data, result

if __name__ == "__main__":
    main()
